test/fixtures/core.scale/label-offset-vertical-axes.json

```diff
@@ -0,0 +1,40 @@
+{
+    "tolerance": "0.008",
```

Comment:
Not sure what to do then :\ Unit tests *must* fail if labels are not correctly positioned, if it doesn't fail with short labels, simply use long labels to make the test relevant. Ideally, tolerance should be `0` (default is `0.001`), so pick the lowest value (with long labels) that passes on Travis but that easily fail if `labelOffset` changes.